docs/operate-and-deploy/changelog.md

```diff
@@ -6,6 +6,12 @@ description: Lists changes to the ksqlDB codebase
 keywords: ksqldb, changelog
 ---
 
+Version 0.17.0
+--------------
+
+- [Announcing ksqlDB 0.17.0](https://www.confluent.io/blog/announcing-ksqldb-0-17-0-new-features-and-updates/)
+- [ksqlDB v0.17.0 changelog](https://github.com/confluentinc/ksql/blob/master/CHANGELOG.md#0170-2021-04-19)
```

Comment:
@vcrfxia for a sanity check - the date should be the day we roll to prod and not the day we release the blog, right/.

Reply:
We don't have this formally written anywhere, but IMO it should be when the public docker images go live. We can merge the PR once we deploy to prod since that's the first time users will be looking for the changes.